docs/hashing.rst

```diff
@@ -30,7 +30,7 @@ Message's creator perspective (:py:func:`~nacl.hash.sha256`,
 
     # define a 1024 bytes log message
     msg = 16*b'256 BytesMessage'
-    digest = nacl.hash.HASHER(msg, encoder=nacl.encoding.HexEncoder)
+    digest = HASHER(msg, encoder=nacl.encoding.HexEncoder)
```

Comment:
Should we switch these codeblocks to doctest? That would prevent this sort of bug from occurring.

Reply:
I completely agree with the plan switching the examples to doctests, but I'd rather fix the line now, instead of waiting for the doctests change.

Reply:
Okay. I'll file an issue to remind us to switch to doctest.